tests/Database/DatabaseEloquentStrictMorphsTest.php

```diff
@@ -39,7 +40,7 @@ public function testStrictModeDoesNotThrowExceptionWhenMorphMap()
 
     public function testMapsCanBeEnforcedInOneMethod()
     {
-        $model = TestModel::make();
+        $model = new TestModel;
```

Comment:
faced issue while running test on single file:

![image](https://user-images.githubusercontent.com/172966/156518766-670dcfff-65a4-48f6-bdb8-fbd8db506a76.png)

Change to `new TestModel` fixed that.